outline.md

```diff
@@ -0,0 +1,59 @@
+To frame the talk, we'll explain the goals of an inner source program as well as the context within-which we've begun to pursue those goals.
+Even though the learning in this presentation is occurring in that context, we suspect that much of what's here applies generally.
+
+When building an inner source program you should have a vision of what services you will provide to facilitate inner sourcing _as well as_ a strategy you will employ to roll out and make effective these services in your organization.
+We'll explain principles, resources, and specific examples that can aid and guide in doing so.
+
+The services of the inner source program should provide two things.
+First, they must teach, define, and document the type of culture that is conducive to inner source.
+Second, there must be software that reinforces that culture while simultaneously lowering the barrier to entry to participating in that culture.
+Examples of these services include:
+
+* Documentation on how to do inner source (and relation of that documentation to [InnerSourcePatterns](http://innersourcecommons.org/patterns)).
+* Software for automated, two-way discovery of inner source projects (maintainer to contributor/user and contributor/user to maintainer).
+* Software for a data-driven understanding of the impact of inner source projects.
+* Software/convention to facilitate individuals communicating with one another.
+* Curated training on the above software and documentation.
+
+The rollout of an inner source program is similar to the rollout of any large, grass-roots initiative.
+Documented patterns in this area (well-summarized in [the Fearless Change series](http://www.fearlesschangepatterns.com/)) apply and work.
+We'll review real examples of how these patterns played out in our journey.
+We'll also share a few patterns unique to inner source rollout including:
+
+* Run your inner source program as inner source projects.
+* Do (or say) nothing save it is documented.
+* Program growth/maturity progression starts with facilitating any synchronous, in-person, side-project talking collaboration then grows to facilitating asynchronous, main-project coding collaboration.
+
+After explaining these principles with clear words and real anecdotes, we'll summarize the current state of our inner source program, where we're hoping it will go, and how we aim to get there. We'll express our hope to learn from and work with everyone present via open dialogue and sharing in the inner source commons.
+
+#### Takeaways
+
+After attending this presentation, we hope that viewers will:
+
+* Understand the suggested purposes and principles of an inner source program.
+* Understand the suggested component parts of an inner source program and their principles.
+* Gain ideas on how to roll out an inner source program.
```

Comment:
I like this one.  I think you should put it first since it has the most immediate value to your audience.

Reply:
Sure.